docs/DeveloperGuide.md

```diff
@@ -335,6 +335,30 @@ Alternatives:
     - Pros: Easy to type
     - Cons: Easier to type the wrong short-form command as they differ by 1 letter
 
+
+
+### Sort Feature
+
+#### Current Implementation
+
+The sort feature allows the user to sort the list of patients, appointments and bills in the application. The sorting is done.
```

Comment:
Don't need to mention the sorting is done.

Reply:
fixed